File: tema/src/main/java/ro/mpp/domain/Flight.java
package ro.mpp.domain;

import java.time.LocalDate;
import java.time.LocalTime;

public class Flight extends Entity<Long>{
    private String destination;
    private LocalDate date;
    private LocalTime hour;
    private String airport;
    private int no_seats;

    public Flight(String destination, LocalDate date, LocalTime hour, String airport, int no_seats) {
        this.destination = destination;
        this.date = date;
        this.hour = hour;
        this.airport = airport;
        this.no_seats = no_seats;
    }

    public String getDestination() {
        return destination;
    }

    public void setDestination(String destination) {
        this.destination = destination;
    }

    public LocalDate getDate() {
        return date;
    }

    public void setDate(LocalDate date) {
        this.date = date;
    }

    public LocalTime getHour() {
        return hour;
    }

    public void setHour(LocalTime hour) {
        this.hour = hour;
    }

    public String getAirport() {
        return airport;
    }

    public void setAirport(String airport) {
        this.airport = airport;
    }

    public int getNo_seats() {
        return no_seats;
    }

    public void setNo_seats(int no_seats) {
        this.no_seats = no_seats;
    }
}

File: Anul_2/Semestrul_2/Medii de proiectare si programare/mpp-proiect-csharp-andrea-muresan/travelAgency/travelAgency/domain/Flight.cs
using System;

namespace travelAgency.domain
{
    public class Flight : Entity<int>
    {
        public string Destination { get; set; }
        public DateTime Date { get; set; }
        public string Airport { get; set; }
        public int NoSeats { get; set; }

        public Flight(string destination, DateTime date, string airport, int noSeats)
        {
            Destination = destination;
            this.Date = date;
            Airport = airport;
            NoSeats = noSeats;
        }
        public Flight(){}

        public Flight(Flight flight)
        {
            this.Destination = flight.Destination;
            this.Date = flight.Date;
            this.Airport = flight.Airport;
            this.NoSeats = flight.NoSeats;
            this.Id = flight.Id;
        }

        public override string ToString()
        {
            return Destination + Date;
        }
    }
}